media-fonts/inconsolata-lgc/inconsolata-lgc-1.1.0_p20140917.ebuild

```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit font
+
+DESCRIPTION="Modified version of Inconsolata Hellenic adding the Cyrillic alphabet"
+HOMEPAGE="https://github.com/DeLaGuardo/Inconsolata-LGC"
+SRC_URI="https://github.com/DeLaGuardo/Inconsolata-LGC/tarball/d086a618248121d61e3f3de64b4301230d1c860c -> ${P}.tar.gz"
+
+LICENSE="OFL-1.1"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~x86"
```

Comment:
I get it's just font but have you tested it on all those arches?

Reply:
This is a direct derivative of the Inconsolata font, and the ebuild for that contained the keywords we see here. As you say, it's just a font; however, if the policy dictates absolutely no nomination of a keyword not tested by the person adding it, even for a TTF font, I will certainly strip this down to `~amd64`.